src/obstacle/MeshObstacle.cxx

```diff
@@ -503,7 +505,8 @@ bool MeshObstacle::containsPointNoOctree(const float point[3]) const
         else if (checkTypes[c] == CheckOutside)
         {
             hasOutsides = true;
-            vec3sub (dir, point, checkPoints[c]);
+            for (int i = 0; i < 3; i++)
+                dir[i] = point[i] - checkPoints[c][i];
```

Comment:
rather than creating these loops, it would have been better to define a local method to do this, so the expressive power of the new version is at least equivalent to the old one.